hack/verify_boilerplate.py

```diff
@@ -154,7 +154,7 @@ def get_regexs():
     # Search for "YEAR" which exists in the boilerplate, but shouldn't in the real thing
     regexs["year"] = re.compile('YEAR')
     # dates can be 2014, 2015, 2016 or 2017, company holder names can be anything
-    regexs["date"] = re.compile('(2014|2015|2016|2017)')
```

Comment:
I wonder if this gives any more value over something more flexible like `\d{4}`...